chromadb/test/test_api.py

```diff
@@ -1614,19 +1613,3 @@ def test_ssl_self_signed_without_ssl_verify(client_ssl):
     )
     client_ssl.clear_system_cache()
     assert "CERTIFICATE_VERIFY_FAILED" in "".join(stack_trace)
-
-
-def test_ssl_self_signed_with_verify_false(client_ssl):
```

Comment:
httpx does not emit warnings when verify is explicitly set to False.

Reply:
does it error or give any indication?

Reply:
no, which imo is more correct behavior